src/components/DisplayToy.js

```diff
@@ -1,35 +1,32 @@
-import React, { useState } from 'react';
+import React from 'react';
 import ReviewsContainer from '../containers/ReviewsContainer';
 import { Container, Button } from 'react-bootstrap';
 import { useDispatch, useSelector } from 'react-redux';
 import { deleteToy } from '../actions/deleteToy.js';
 import { selectToy } from '../selectors/toySelectors';
+import { useHistory } from 'react-router';
 
 const DisplayToy = (props) => {
-  const [display, setDisplay] = useState(true);
-  const [show, setShow] = useState(false);
   const dispatch = useDispatch();
 
   const toy = useSelector(selectToy(props.match.params.id));
 
+  const history = useHistory();
+
   const handleDelete = (toyId) => {
-    dispatch(deleteToy(toyId));
-    setDisplay(false);
-    setShow(true);
+    dispatch(deleteToy(toyId, history));
   };
```

Comment:
How about just call `dispatch(...)` from the `onClick` since we're not doing anything else in `handleDelete`?

Reply:
For sure!